aten/src/ATen/Declarations.cwrap

```diff
@@ -2747,7 +2747,6 @@
     - CPU
     - CUDA
   variants:
-    - method
```

Comment:
Why did we remove the method variant?

Reply:
The method variant is not used, hence the change. This also only concerns TH/THC and not ATen, so I thought I’d remove it.